redbeams/src/main/java/com/sequenceiq/redbeams/configuration/DatabaseServerSslCertificateConfig.java

```diff
@@ -246,58 +289,72 @@ public SslCertificateEntry getCertByPlatformAndVersion(String cloudPlatform, int
         }
     }
 
-    public SslCertificateEntry getCertByPlatformAndCloudProviderIdentifier(String cloudPlatform, String cloudProviderIdentifier) {
+    private <V> V getValueByCloudPlatformAndRegion(Map<String, V> valuesByCloudPlatformMap, String cloudPlatform, String region) {
+        String cloudPlatformNormalized = cloudPlatform.toLowerCase();
+        V result = null;
```

Comment:
remove this null initialization please, and add `else` to each `if` and `return`-s like
```
if() {
...
return <>;
} else if () {
...
return <>;
} else {
...
return <>;
}
```
so the code indentation would reflect how the logic works

Reply:
Cmon, it's a style thing that one return or multiple. We shouldn't review this manner of the code, I think.

Reply:
I think code readability matters a lot to see what are the exit points without going through the whole method. When you are looking at one method it maybe doesn't matter, but when you are going through a lot for a review or for investigating some bug/incident, it's much more easy when the code is readable. I wouldn't request a change for this thing only, but if there is something else to change, then straightening these out is not a much of effort and later helps a lot.

Reply:
It's a style thing. I like one return in a method, I think it is even easier to understand the logic that way. Multiple returns end up with multiple exit points which harder to be followed.
But that's my preference the other one is yours, I think we should not review the code from this manner during code-reviews, because it restricts our freedom in coding which is a bad habbit, probably there would be CS rule to restrict this if all the developers would be on the same page in this topic.

Reply:
The variable has been kept, renamed (`globalValue`) and initialized with the global map entry value. Otherwise it now follows the above convention.